features/support/publishing_cuke_helpers.rb

```diff
@@ -26,6 +26,8 @@ def make_post(text)
   def submit_publisher
     txt = find("#publisher #status_message_text").value
     find("#publisher .btn-primary").click
+    # wait for the publisher to be closed
+    find("#publisher")["class"].should include("closed")
```

Comment:
This is taken from `https://github.com/diaspora/diaspora/blob/55fee86a95665446b197e16606be81378664e773/features/step_definitions/custom_web_steps.rb#L72` and I confirm that even if the page is loaded with the publisher open, the `closed` class is added to it when the post is submitted (this test would fail otherwise, but it was green with Ruby 2.3 and locally).

I thought about removing the popovers and I'll try that if it helps, however this changes the test behavior, it means we do not test anymore that the publisher can be submitted even if the popovers are there.